src/config/lang/english.go

```diff
@@ -37,6 +37,7 @@ const (
 	RootCmdFlagNoProgress  = "Disable fancy UI progress bars, spinners, logos, etc"
 	RootCmdFlagCachePath   = "Specify the location of the Zarf cache directory"
 	RootCmdFlagTempDir     = "Specify the temporary directory to use for intermediate files"
+	RootCmdFlagInseure     = "Allow access to insecure registries and disable other recommended security enforcements. This flag should only be used if you have a specific reason and accept the reduced security posture."
```

Comment:
Not calling this out in the requested changes, but wanted to add a comment for @Madeline-UX to look at this verbiage to make sure it makes sense. `--insecure` here is doing two things:

1. Allowing connections to `http` image registries on `create` or `deploy`
2. Disabling Transport Layer Security (TLS) verification for connections to `https` registries

Technically both of these apply only to image registries at the moment and I do feel this message may require more balancing between `create` personas (who know what http/tls are) and `deploy` personas who may not.

Reply:
Yeah I agree, I just left it generic as "other security enforcements" because of that.

Reply:
Would it be better to specify which security enforcements it disables (the TLS)? That way, it's specific and can be researched if needed. 

I am concerned that the vagueness of "and other security enforcements" may keep users from being able to accurately assess the risk.  Unless we are going for scary - vague. Don't do this ... unless you really need to.

Reply:
@Racer159 @jeff-mccoy I could do a quick async UX test of the language - I need to know what we are wanting to test though. 

We assume that the creator understands and knows what these terms mean
We are not sure if the deploy persona "Ashton does?"